### PR TITLE
[ci] enable Qt6 builds and remove azure pipelines

### PR DESCRIPTION
**Pull request by 375gnu**
i don't think it makes sense to use two different CI systems

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -85,14 +90,40 @@ jobs:
...
+        - os: ubuntu-22.04
+          avif: ON
+          gles: ON
+          glut: ON
+          gtk3: ON
+          minaudio: ON
+        - os: macos-latest
+          avif: ON
+          gles: OFF     # TODO: Figure out how to enable
+          glut: OFF     # TODO: Figure out how to enable
```
Comment (levinli303) [inline .github/workflows/ci.yml:114]: glut can be turned on I think? I remember I could build it some time ago.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -85,14 +90,40 @@ jobs:
...
+          minaudio: OFF # Requires newer g++
+        - os: ubuntu-22.04
+          avif: ON
+          gles: ON
+          glut: ON
+          gtk3: ON
+          minaudio: ON
+        - os: macos-latest
+          avif: ON
+          gles: OFF     # TODO: Figure out how to enable
```
Comment (levinli303) [inline .github/workflows/ci.yml:113]: only the iOS-on-Mac apps can use gles. Mac apps cannot access GLES APIs, so no need for this TODO

**Review by levinli303 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (375gnu) [reply .github/workflows/ci.yml:114]: then i need your help, i tried to enable it first but the build finished with errors (perhaps glut library was not available to the linker)

Comment (375gnu) [reply .github/workflows/ci.yml]: fixed this

**Comment by 375gnu** [comment]
@levinli303 any other changes to make?

**Review by levinli303 — APPROVED** [review]

**Pull request merged**